docs/PTFUserGuide.md

```diff
@@ -246,19 +246,19 @@ For example, if a user has the following configuration in the PTFconfig
 
 __Examples__
 
-1.   Command “set ptfprop” can be used in the script to display all properties in the PTFconfig. User will get the outputs like:
+1.   Command “set ptfprop" can be used in the script to display all properties in the PTFconfig. User will get the outputs like:
```

Comment:
The first quotation mark should be changed too.
There're also many other places that need to replace the first quotation mark.